### DISCUSSION TITLE
Markdown editor inserts \xa0 non-breaking spaces

### DISCUSSION THREAD

**Discussion by FabriceSalvaire** [requarks/wiki#6320] [Error / Bug Report]
I use an external editor to edit the markdown content using the Firefox extension withExEditor. And I noticed the editor uses \xa0 non-breaking spaces when there is more than one space in the text, e.g.:

```
* a bullet
  \xa0some text
```

It makes the markdown unreadable and **can spoil a Markdown table syntax** in some case.

I understand non-power user want a user-friendly Markdown editor, but it is the first I have a bad experience with a Markdown Textarea in a browser (issues with spell checker, cursor that disappear, \xa0 spam, error message when typing a link due to incomplete syntax...).

**Comment by NGPixel** [comment]
Very unlikely that this is added by Wiki.js... Are you sure that's not the extension you're using? I can't replicate what you see.
